fastlane/Fastfile

```diff
@@ -172,16 +176,49 @@ platform :ios do
   #####################################################################################
   desc 'Creates a new release branch from the current trunk'
   lane :code_freeze do |options|
-    ios_codefreeze_prechecks(options)
+    # Verify that there's nothing in progress in the working copy
+    ensure_git_status_clean
+
+    # Check out the up-to-date default branch, the designated starting point for the code freeze
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    message = <<-MESSAGE
+
+      Code Freeze:
+      • New release branch from #{DEFAULT_BRANCH}: release/#{release_version_next}
+      • Current release version and build code: #{release_version_current} (#{build_code_current}).
+      • New release version and build code: #{release_version_next} (#{build_code_code_freeze}).
+
+    MESSAGE
+
+    UI.important(message)
+    UI.user_error!('Aborted by user request') if !options[:skip_confirm] && !UI.confirm('Do you want to continue?')
+
+    # Create the release branch
+    UI.message('Creating release branch...')
+    ensure_git_branch(branch: DEFAULT_BRANCH)
```

Comment:
How do you feel about removing this check? What are the chances the branch changed under the hood after the `checkout_and_pull` command a few lines above?

```suggestion
```